Dockerfile

```diff
@@ -18,14 +18,8 @@ ARG REDIS_IMAGE=${REDIS_IMAGE}
 ARG PAUSE_IMAGE=${PAUSE_IMAGE}
 ARG SQLITE_IMAGE=${SQLITE_IMAGE}
 
-ENV GOROOT=/usr/local/go
-ENV PATH=$GOPATH/bin:$GOROOT/bin:$PATH 
-
-# Get Go binary
-RUN curl -o go1.21.13.linux-amd64.tar.gz https://dl.google.com/go/go1.21.13.linux-amd64.tar.gz
-RUN tar -xzf go1.21.13.linux-amd64.tar.gz  &&\
-    mv go /usr/local
-
+# Switch to root to copy files and work
+USER root
 COPY . /cli
```

Comment:
if you need root for the copy, can you switch back to a regular user after the copy is done